webvtt/rendering/cues-with-video/processing-model/regions/support/scroll_up.vtt

```diff
@@ -0,0 +1,16 @@
+WEVTT
+
+REGION
+id:1
+lines:2
+regionanchor:0%,0%
+scroll:up
+
+00:00:00.000 --> 00:00:05.000 region:1
+This is a first test subtitle
+
+00:00:01.000 --> 00:00:05.000 region:1
```

Comment:
Doesn't this cue (and the one below) need to start at 00:00:00.000 to show up when `takeScreenshot()` runs? Or should the test use `takeScreenshotDelayed(2000)`? Or am I missing something here? :-)